### PR TITLE
Joshua Machine Translation Toolkit Homewbrew Formula

### PR DESCRIPTION
**Pull request by lewismc**
Hi Folks,
[Joshua Decoder](http://joshua-decoder.org/) is an open-source statistical machine translation decoder for phrase-based, hierarchical, and syntax-based machine translation, written in Java.
We would very like for a brew formula to be available for people to use.
The tests included simply verify the unit test suite provided by Joshua.
Thanks

### CODE REVIEW

**Comment by lewismc** [comment]
Folks, any idea how I can squash all of this to clean up the history? Sorry about the mess here.

File: Library/Formula/joshua-v.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  resource "ar-en-phrase-pack" do
+    url "http://cs.jhu.edu/~post/language-packs/language-pack-ar-en-phrase-2015-03-18.tgz"
+    sha256 "2B6665B58B11E4C25D48191D3D5B62B7C591851A9767B14F9CCEBF1951FDDF90"
+  end
+
+  def install
+
+    system "ant"   
+    
+    system "ant javadoc"
```
Comment (dunn) [inline Library/Formula/joshua-v.rb]: The arguments to `system` should be comma-separated: `system "ant", "javadoc"`.  Same below in the test. Thanks!

File: Library/Formula/joshua-v.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      resource("ar-en-phrase-pack").stage { mv "language-pack-ar-en-phrase-2015-03-18", share/"joshualanguagepacks" }
+    elsif build.with? "es-en-phrase-pack"
+      resource("es-en-phrase-pack").stage { mv "language-pack-es-en-phrase-2015-03-06", share/"joshualanguagepacks" }
+    elsif build.with? "ar-en-phrase-pack"
+      resource("ar-en-phrase-pack").stage { mv "language-pack-ar-en-phrase-2015-03-18", share/"joshualanguagepacks" }
+    end
+  end
+
+  test do    
+    system "ant test"
```
Comment (dunn) [inline Library/Formula/joshua-v.rb]: The test is run in a temporary directory separate from the build tree, so it won't have access to the source files.

**Comment by bfontaine** [comment]
@lewismc Where do these commit come from? They don’t seem related to Homebrew. How did you make this pull-request?

File: Library/Formula/joshua-v.rb (hunk 2 above)
Comment (lewismc) [inline Library/Formula/joshua-v.rb]: Hi @dunn, can you please advise a solution here?

**Comment by lewismc** [comment]
Closing this PR in favor of #44384. That one is cleaner.

**Pull request closed without merging**